tests/observability/helper/helper.go

```diff
@@ -0,0 +1,167 @@
+package helper
```

Comment:
Please let's keep packages namespaced. Rename ot to observabilityhelper. As far as I understand this is best practice. Usually it's not good to have multiple packages with the same name across the project.

Reply:
Where can I find a doc/statement about this best practice regarding package naming of a module? At least in the official golang docs I don't see any mention to that constraint: https://go.dev/doc/effective_go#names

> The package name is only the default name for imports; **it need not be unique across all source code**, and in the rare case of a collision the importing package can choose a different name to use locally. In any case, **confusion is rare because the file name in the import determines just which package is being used**.
> 
> Another convention is that the package name is the base name of its source directory; the package in src/encoding/base64 is imported as "encoding/base64" but has name base64, not encoding_base64 and not encodingBase64.

Reply:
Well, yes packages can have the same name in the project however what value it brings?
For my perspective nethelper, globalhelper much more descriptive than helper because I can understand from the name prefix a location of the function. 
One more issue is importing 2 packages with the same name because It requires additional import tweaks.

Reply:
It's just for not repeating words when it's not necessary because of the context.

![image](https://user-images.githubusercontent.com/87083379/171851410-9ca97f3f-3b16-41b1-9c9a-54f952f20f5a.png)
vs
![image](https://user-images.githubusercontent.com/87083379/171851507-ea0608eb-8551-405f-99c5-74645f197216.png)

Reply:
Ok, I agree. +1
Let's do the following steps. 

On the first step please, re-name you package to affiliatedcerthelper and affiliatedcertparameters in the PR.

On the second step add PR which'll rename all [prefix]helpers and [prefix]parameters across the entire project in to helper(on global lever) and tshelper (on test suite level). Same for parameters packages.

What do you think?

Reply:
Follow up PR: #99